gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -240,56 +241,56 @@ class GateConfig extends RedisHttpSessionConfiguration {
 
       List<ServiceSelector> selectors = []
       endpoints.each { sourceApp, url ->
-        def service = buildService(okHttpClient, ClouddriverService, newFixedEndpoint(url))
+        def service = buildService("clouddriver",clientProvider, ClouddriverService, newFixedEndpoint(url))
```

Comment:
```suggestion
        def service = buildService("clouddriver", clientProvider, ClouddriverService, newFixedEndpoint(url))
```
not sure how this snuck past spotless, maybe run a spotlessApply manually and see if it fixes it?